Sage/carbon
# fix(action-popover): fix issue with stories not rendering in storybook - FE-6264

### Proposed behaviour

Ensure stories render in Storybook

### Current behaviour

Stories are missing from Storybook

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

- Ensure that `ActionPopover` renders in the sidebar of Storybook.
- Ensure that all `ActionPopover` examples render.
